skill-tree /src/test/java/com/RDS/skilltree/TestContainerManger.java

```diff
@@ -0,0 +1,24 @@
+package com.RDS.skilltree;
+
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MySQLContainer;
+
+abstract class TestContainerManger {
+    @ServiceConnection
```

Comment:
Does this annotation do anything? Since the container is created separately anyway?

Reply:
Yes it does work, its dependent on the Mysql container.